lib/openingScreen.dart

```diff
@@ -0,0 +1,111 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+import 'menu.dart';
+
+class openingScreen extends StatefulWidget {
```

Comment:
This file should be placed under the `lib/view/screens/onboarding` folder, and inside the `onboarding_screen.dart` file.

Also rename this class to `OnBoardingScreen`